diff --git a/node_modules/@minka/bridge-sdk/src/server/controllers/status.controller.js b/node_modules/@minka/bridge-sdk/src/server/controllers/status.controller.js
index 6a6e37f..47b5fc3 100644
--- a/node_modules/@minka/bridge-sdk/src/server/controllers/status.controller.js
+++ b/node_modules/@minka/bridge-sdk/src/server/controllers/status.controller.js
@@ -13,12 +13,17 @@ exports.StatusController = void 0;
 const inversify_1 = require("inversify");
 const routing_controllers_1 = require("routing-controllers");
 let StatusController = class StatusController {
-    status() {
-        return { status: 'ok' };
+    status(polling) {
+        if( polling === "enable" || polling === "disable" ){
+            const updated = polling === "enable";
+            global.blockchainPolling = updated;
+        }
+        return { status: 'ok', blockchainPolling: global.blockchainPolling || false };
     }
 };
 __decorate([
     (0, routing_controllers_1.Get)(),
+    __param(0, (0, routing_controllers_1.QueryParam)('polling')),
     __metadata("design:type", Function),
     __metadata("design:paramtypes", []),
     __metadata("design:returntype", void 0)
